### PR TITLE
[BXMSPROD-1441] cron meta nightly PRODUCT_VERSION upgraded to 7.12.0

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 
https://issues.redhat.com/browse/BXMSPROD-1441

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by radtriste** [comment]
This will be once we branch the 7.12 product. For now `main` branch contains `main` UMB_VERSION

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ pipeline{
                         [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly-with-upstream'],
                         [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "${RHBA_VERSION_PREFIX}"],
                         [\$class: 'StringParameterValue', name: 'KOGITO_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-kogito-main/content-compressed'],
-                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
                         [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
                         [\$class: 'StringParameterValue', name: 'OPTAPLANNER_PRODUCT_VERSION', value: '8.6.0'],
```
Comment (radtriste) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:44]:
```suggestion
                        [\$class: 'StringParameterValue', name: 'OPTAPLANNER_PRODUCT_VERSION', value: '7.12.0'],
```

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 2]
```diff
@@ -39,7 +39,7 @@ pipeline{
                         [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly-with-upstream'],
                         [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "${RHBA_VERSION_PREFIX}"],
                         [\$class: 'StringParameterValue', name: 'KOGITO_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-kogito-main/content-compressed'],
-                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
```
Comment (radtriste) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:42]:
```suggestion
                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
```

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 3]
```diff
@@ -53,8 +53,8 @@ pipeline{
             steps {
                 build job: 'kogito-tooling.nightly/main', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-main/content-compressed'],
-                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.11.0'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
```
Comment (radtriste) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:56]:
```suggestion
                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
```

**Review by mbiarnes — APPROVED** [review]
LGTM

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
I guess the PR title could be changed

**Comment by Ginxo** [comment]
@mbiarnes @radtriste please merge if you are ok with this

**Pull request merged**